contracts/proxy/utils/Initializable.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
 // OpenZeppelin Contracts (last updated v4.9.0) (proxy/utils/Initializable.sol)
 
-pragma solidity ^0.8.19;
+pragma solidity ^0.8.20;
```

Comment:
Why do we go to 0.8.20 here ?

Reply:
`@custom` in structs is only available in 0.8.20. We need to do this in the whole library.